patches/minecraft/net/minecraft/world/WorldProvider.java.patch

```diff
@@ -256,6 +256,16 @@
 +        return null;
 +    }
 +
++    /**
++     * Called from {@link net.minecraft.world.chunk.Chunk#checkLight()}, to get additional blocks in radius for checking.
++     * Confused? See issue MinecraftForge#4723
++     * @return Additional amount of chunks by amount of blocks required before checking light. Should be in increments of 16.
```

Comment:
Instead of forcing the implementor to return multiples of 16, why not just have the return value be in chunks, and multiply by 16 in the patch code?

Reply:
There isn't really a reason to have the return be an `int` here. Values less than 16 won't prevent the issue, and any greater value gives no additional benefit.